app/src/main/java/ie/teamchile/smartapp/util/AdapterSpinner.java

```diff
@@ -43,20 +41,25 @@ public AdapterSpinner(Context context, int arrayResource, int layoutresource, in
 
     @Override
     public View getDropDownView(int position, View convertView, ViewGroup parent) {
-        convertView = View.inflate(context, layoutresource, null);
-        TextView tvSpinnerItem = (TextView) convertView.findViewById(tvResource);
-        tvSpinnerItem.setText(list.get(position));
+        ViewHolder holder;
+        if (convertView == null) {
+            convertView = View.inflate(context, layoutresource, null);
+            holder = new ViewHolder(convertView);
+            convertView.setTag(holder);
+        } else
+            holder = (ViewHolder) convertView.getTag();
+
+        holder.tvSpinnerItem.setText(list.get(position));
```

Comment:
At 52. No its still a child of the getDropDownView method, which is a child of the main class so it need the double indent.